hkl/tests/test_tardis.py

```diff
@@ -1,43 +1,16 @@
-from hkl import Constraint
-from hkl import E6C, SimMixin
-from hkl import Lattice
-from ophyd import Component as Cpt
-from ophyd import SoftPositioner
-import hkl.calc
 import numpy as np
 import numpy.testing
 import pytest
+from ophyd import Component as Cpt
+from ophyd import SoftPositioner
 
-
-TARDIS_TEST_MODE = "lifting_detector_mu"
-
-
-class Tardis(SimMixin, E6C):
-    # theta
-    theta = Cpt(SoftPositioner, init_pos=0)
-    omega = Cpt(SoftPositioner, init_pos=0)
-    chi = Cpt(SoftPositioner, init_pos=0)
-    phi = Cpt(SoftPositioner, init_pos=0)
-    # delta, gamma
-    delta = Cpt(SoftPositioner, init_pos=0)
-    gamma = Cpt(SoftPositioner, init_pos=0)
-
-
-@pytest.fixture(scope="function")
-def tardis():
-    tardis = Tardis("", name="tardis")
-    tardis.calc.engine.mode = TARDIS_TEST_MODE
-    # re-map Tardis' axis names onto what an E6C expects
-    tardis.calc.physical_axis_names = {
-        "mu": "theta",
-        "omega": "omega",
-        "chi": "chi",
-        "phi": "phi",
-        "gamma": "delta",
-        "delta": "gamma",
-    }
-    tardis.wait_for_connection()
-    return tardis
+from .. import E6C
+from .. import Constraint
+from .. import SimMixin
+from .. import calc as hkl_calc
+from ..util import new_lattice
+from .__init__ import TARDIS_TEST_MODE
+from .__init__ import new_sample
```

Comment:
Have you already tried this without the `__init__` qualifier?

```suggestion
from . import TARDIS_TEST_MODE
from . import new_sample
```

The current approach works for now, but importing from `.__init__` could lead to some circular import traps down the line. In this situation, I might instead create a util.py or _util.py file (in the tests/ directory, here) that defines `TARDIS_TEST_MODE` and `new_sample`. 

Then you could call `from _util import TARDIS_TEST_MODE, new_sample` within __init__.py, test_configuration.py, test_tardis.py, etc.

[This SO post](https://stackoverflow.com/a/42862399) suggests that with the helper module, you could also use the `from . import new_sample` format.

Reply:
eef187f9950353add38523608eea8aa21049e0f9, skipped the helper module for now